scripts/feature_tests/upgrade/upgrade_vars.sh

```diff
@@ -28,14 +28,14 @@ function get_latest_capi_release() {
 }
 
 # CAPI release version which we upgrade from.
-export CAPIRELEASE="v0.4.0"
+export CAPIRELEASE="v0.4.1"
 CAPI_REL_TO_VERSION="$(get_latest_capi_release)" || true
 # CAPI release version which we upgrade to.
 export CAPI_REL_TO_VERSION
 
-export FROM_K8S_VERSION="v1.21.1"
+export FROM_K8S_VERSION="v1.21.2"
 export KUBERNETES_VERSION=${FROM_K8S_VERSION}
-export UPGRADED_K8S_VERSION="v1.21.2"
+export UPGRADED_K8S_VERSION="v1.22.0"
```

Comment:
what about feature tests k8s vars? https://github.com/metal3-io/metal3-dev-env/blob/346594f241c5f7d1e4bc1bc659f5416352b209ab/scripts/feature_tests/node_reuse/node_reuse.sh#L11-L13

Reply:
Because feature tests will be sourcing this file while running, whereas upgrade sources another(upgrade_vars.sh). This vars used to be not here previously and feature tests (to be specific node reuse only) were using whatever it is set in lib/common.sh.

Reply:
Never a fan of putting the same thing in two places. Anyways I ll add it. Good catch that I missed it.

Reply:
Done @furkatgofurov7